maven-enforcer-plugin/src/main/java/org/apache/maven/plugins/enforcer/EnforceMojo.java

```diff
@@ -187,7 +187,7 @@ public void execute()
                 // store the current rule for
                 // logging purposes
                 currentRule = rule.getClass().getName();
-                log.debug( "Executing rule: " + currentRule );
+                log.info( "Executing rule: " + currentRule );
```

Comment:
Rule is executed in line 199, we will have such message even when rule will not be executed in case of cache.

Is it what we want?